src/db/oracledb/meal-plans-dao.js

```diff
@@ -32,4 +32,27 @@ const getMealPlansByOsuId = async (osuId, query) => {
   }
 };
 
-export { getMealPlansByOsuId };
+/**
+ * Queries data source for raw meal-plan data
+ *
+ * @param {string} osuId OSU ID of person to select addresses from
+ * @param {string} mealPlanId ID of meal plan to select
+ * @returns {Promise<object>} Serialized address resource
+ */
+const getMealPlanByMealPlanId = async (osuId, mealPlanId) => {
+  const connection = await getConnection('ods');
+  try {
+    const query = { osuId, mealPlanId };
+    const { rows } = await connection.execute(contrib.getMealPlansByOsuId(query), query);
+
+    if (rows.length < 1) {
+      return undefined;
+    }
+
+    return rows[0];
```

Comment:
Two ways to improved:

```js
return rows.length < 1 ? undefined : rows[0];
```

or just return `rows[0]` if `row.length >= 1`. Return undefined if not necessary.

```js
if (rows.length >= 1) {
  return rows[0];
}
```

I personally prefer the first one since the second one is too implicitly to me.

Reply:
I agree the first one is better